tests/unit/customizations/s3/test_subcommands.py

```diff
@@ -356,6 +357,28 @@ def test_run_cp_put(self):
         output_str = "(dryrun) upload: %s to %s" % (rel_local_file, s3_file)
         self.assertIn(output_str, self.output.getvalue())
 
+    @patch('sys.stdin', new_callable=StringIO)
+    def test_run_cp_streaming_put(self, mock_stdin):
+        s3_file = 's3://' + self.bucket + '/' + 'text1.txt'
+        local_file = self.loc_files[0]
+        mock_stdin.write(open(local_file).read())
+        filters = [['--include', '*']]
+        params = {'dir_op': False, 'dryrun': True, 'quiet': False,
+                  'src': '-', 'dest': s3_file, 'filters': filters,
+                  'paths_type': 'locals3', 'region': 'us-east-1',
+                  'endpoint_url': None, 'verify_ssl': None,
+                  'follow_symlinks': True, 'page_size': None,
+                  'is_stream': True}
+        config = RuntimeConfig().build_config(
+            multipart_threshold=1, multipart_chunksize=2)
+        cmd_arc = CommandArchitecture(
+            self.session, 'cp', params, runtime_config=config)
+        cmd_arc.create_instructions()
+        self.patch_make_request()
+        cmd_arc.run()
+        output_str = "Completed 1 part(s) with ... file(s) remaining"
+        self.assertIn(output_str, self.output.getvalue())
+
```

Comment:
@JordonPhillips the output of this test varies depending on whether we're using Python 2 or 3, which suggests to me that it's not a useful test. I'm at a loss to test that a streaming cp is done using the configured chunk size, short of running it against a real AWS bucket and capturing debug logging.